### PR TITLE
Add snaplet guide to the integrations section

### PR DESCRIPTION
**Pull request by sudoramz**
### Environment parity in Supabase using -----> **Snaplet**

Snaplet is an awesome developer tool that enables you to copy your production database and paste it into your development environment, all whilst sanitising the data and anonymising all personal identifiable information for you.

snappy-holding-supabase-logo

### About the PR
This PR is for an update to the Supabase documentation and it includes instructions on how to do this! Let me know what your thoughts are.

### CODE REVIEW

File: web/docs/guides/integrations/snaplet.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+![connect-your-database](/img/guides/integrations/snaplet/connect-your-database.png)
+
+Enter the credentials of your production Supabase project. Find the "Connection string" in Supabase via `Organization > Project > Settings > Database > Connection string` (at the bottom of the page).
+
+The password is the same password you used when creating the Supabase project.
+
+![supabase-connection-db-info](/img/guides/integrations/snaplet/supabase-connection-db-info.png)
+
+You’ll have to confirm providing Snaplet access to your database. Snaplet will prompt you to only provide `read-only` access to your database. Snaplet has a guide in their documentation on how to to do so [here](https://docs.snaplet.dev/postgresql/create-read-only-role).
```
Comment (Isaiah-Hamilton) [inline web/docs/guides/integrations/snaplet.mdx:49]:
```suggestion
You’ll have to confirm providing Snaplet access to your database. Snaplet will prompt you to only provide `read-only` access to your database. Snaplet has a guide in their documentation on how to do so [here](https://docs.snaplet.dev/postgresql/create-read-only-role).
```

File: web/docs/guides/integrations/snaplet.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+3. Enter your project details
+4. Wait for the new database to launch
+
+> Remember the password you use when creating the project. You’ll need this password to connect your database to Snaplet later.
+
+### 3.2. Make your `postgres` user a superuser
+
+Snaplet requires the ability to drop the database schemas whilst restoring a snapshot. In order to do that, Snaplet requires `superuser` privileges.
+
+1. Navigate to your the `SQL Editor` in your Supabase console
```
Comment (Isaiah-Hamilton) [inline web/docs/guides/integrations/snaplet.mdx:86]:
```suggestion
1. Navigate to the `SQL Editor` in your Supabase console
```

File: web/docs/guides/integrations/snaplet.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+4. Paste your access token in the terminal
+
+### 3.4. Restore to the data target
+
+You're now ready to restore your production snapshot into your Supabase development project.
+
+1. Navigate to your project directory
+2. Run `snaplet setup` – you will be prompted to enter your database credentials. These are the database credentials of your **data target.** This could be your staging or development database
+
+Once you have successfully completed the above steps, you will have be presented with a list of options – these options are databases that are connected to your Snaplet account.
```
Comment (Isaiah-Hamilton) [inline web/docs/guides/integrations/snaplet.mdx:105]:
```suggestion
Once you have successfully completed the above steps, you will have to be presented with a list of options – these options are databases that are connected to your Snaplet account.
```

File: web/docs/guides/integrations/snaplet.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+1. Select a data source from the list
+2. Run `snaplet restore`
+
+## All done!
+
+As a Supabase user, you can see how this solves an issue developers all typically experience when attempting to create multiple development environments and populating each of those environments with data that can be worked with. Snaplet simplifies this process down to creating the respective Supabase projects, connecting the data source (The production database) to Snaplet and telling Snaplet where to restore that data (staging and development databases).
+
+If you want to learn more about Snaplet, you can explore the Snaplet [docs](https://docs.snaplet.dev/). If you have any questions, feel free to [reach out on Discord](https://discord.com/invite/6HUuajc866) .
```
Comment (Isaiah-Hamilton) [inline web/docs/guides/integrations/snaplet.mdx:114]:
```suggestion
If you want to learn more about Snaplet, you can explore the Snaplet [docs](https://docs.snaplet.dev/). If you have any questions, feel free to [reach out on Discord](https://discord.com/invite/6HUuajc866).
```

**Review by Isaiah-Hamilton — DISMISSED** [review]

File: web/docs/guides/integrations/snaplet.mdx (hunk 3 above)
Comment (thorwebdev) [inline web/docs/guides/integrations/snaplet.mdx:105]:
```suggestion
Once completed, you will be presented with a list of databases that are connected to your Snaplet account.
```

**Review by thorwebdev — APPROVED** [review]
LGTM, thanks

**Pull request merged**